examples/mnist/mnist_lib_test.py

```diff
@@ -17,7 +17,7 @@
 
 from absl.testing import absltest
 
-import train
+from examples.mnist import train
```

Comment:
I believe that with this change, we lose the property that "you can easily just fork an example and use it independently" -- how would tests work in that repo? (This code would no longer work, I think)

Reply:
Yeah, that’s right; one benefit of the existing script is it allows each example to be fully self-contained. If someone were to copy just a single example directory to somewhere outside the repo, Python would get confused by `example.`. I’ll have a think if there’s a way around this issue, but nothing comes immediately to mind.